src/config/services/user-services.md

```diff
@@ -1,14 +1,14 @@
 # Per-User Services
 
-Sometimes it would be nice to have user-specific runit services. Services that,
-for example, open an ssh tunnel as your current user, run a virtual machine, or
+Sometimes it can be nice to have user-specific runit services. For example, you
+might want to open an ssh tunnel as the current user, run a virtual machine, or
 regularly run daemons on your behalf. The most common way to do this is to ask a
 system-level service to start a
 [runsvdir(8)](https://man.voidlinux.org/runsvdir.8) service as your user for
```

Comment:
s/for/via/

Reply:
The user services themselves will run via a personal services directory, but we need the system-level service **for** the user's services directory.

Reply:
Fair enough, but still, i find the structure of this sentence quite awkward. Not enough to make it a blocker, though, so let's leave it as-is for now.

Reply:
@flexibeast I tried to make a few changes, what do you think?

Reply:
@ericonr: Oh that's much better, thanks!